test/verify/check-service

```diff
@@ -0,0 +1,129 @@
+        # avoid duplecate result file
+        if not cov_out_file.is_file():
+            cov_out_file.write_text(str_coverage)
+
+    def setUp(self):
+        # remove all blueprints configured in vm.install
+        # stop and disable osbuild-composer.socket service
+        super().setUp()
+        self.allow_journal_messages("http:///run/weldr/api.socket/api/v0/.* couldn't connect: Connection refused")
+
+        self.machine.execute("""
+            for bp in $(composer-cli blueprints list); do
+                composer-cli blueprints delete $bp
+            done
+            systemctl disable --now osbuild-composer.socket
+        """)
+
+        self.addCleanup(self.machine.execute, """
+            for bp in $(composer-cli blueprints list); do
+                composer-cli blueprints delete $bp
+            done
+            for toml_file in /home/admin/files/*.toml; do
+                composer-cli blueprints push $toml_file
+            done
+        """)
+
+    def testBasic(self):
+        b = self.browser
+
+        self.login_and_go("/composer", authorized=True)
+        b.wait_present("#main")
+
+        # start and enable osbuild-composer from UI
+        b.click("button:contains('Start')")
+        # work around .blank-slate-pf element disappear in one second
+        b.wait_in_text("#main", "No Blueprints")
+        b.wait_text(".blank-slate-pf h1", "No Blueprints")
+
+        # create blueprint with random name and description
+        name_template = string.ascii_letters + string.digits + "-" + "_"
+        bp_name = "".join(random.sample(name_template, 20))
+        desc_template = string.ascii_letters + string.digits + " "
+        bp_desc = "".join(random.sample(desc_template, 40))
+        b.click(".blank-slate-pf button:contains('Create Blueprint')")
+        b.wait_present("div[role=dialog] #cmpsr-modal-crt-blueprint")
+        # blueprint name
+        b.set_input_text("#textInput-modal-markup", bp_name)
+        # blueprint description
+        b.set_input_text("#textInput2-modal-markup", bp_desc)
+        # create
+        b.click("#create-blueprint-modal-create-button")
+        b.wait_not_present("div[role=dialog] #cmpsr-modal-crt-blueprint")
+
+        # wait for all available compontents visible
+        with b.wait_timeout(300):
+            b.wait_present("ul[data-list=inputs]")
+        # search for openssh-server pacakge
+        b.set_input_text("#cmpsr-blueprint-input-filter", "openssh-server")
+        b.key_press("\r")
+        with b.wait_timeout(120):
+            b.wait_text("#openssh-server-input", "openssh-server")
+        # add it to blueprint
+        b.click("li[data-input=openssh-server] .fa-plus")
+        # wait for package added
+        b.wait_present("#blueprint-tabs")
+        b.wait_present("li[data-component=openssh-server]")
+        # click commit button
+        b.click(".cmpsr-header__actions .btn-primary")
+        b.wait_visible("#cmpsr-modal-pending-changes")
```

Comment:
`click()` does `wait_visible`. So this command should not be needed as you do `click` as the next command.

Reply:
make sense to me. Thanks!